src/open_inwoner/scss/components/Card/Card.scss

```diff
@@ -11,7 +15,7 @@
   border: var(--card-size-border) solid var(--card-color-border);
   border-radius: var(--border-radius);
   box-sizing: border-box;
-  display: flex;
+  // display: flex;
```

Comment:
Not sure if we can just remove this without it causing problems elsewhere? @jiromaykin